services/backend/src/router/profile/profile.js

```diff
@@ -17,7 +17,8 @@ profileRouter
     [UUIDValidator, langValidator],
     profile.getPublicProfileById
   )
-  .put(keycloak.protect(), updateProfileValidator, profile.updateProfile);
+  .put(keycloak.protect(), updateProfileValidator, profile.updateProfile)
+  .post(keycloak.protect(), updateProfileValidator, profile.updateProfile);
```

Comment:
we should probably not add a post to profile, since Users and Profiles are now the same "thing", and we create a user early on, so we should only update that profile

Reply:
Changed code to use put instead of post